transforms/language/doc_quality/ray/Makefile

```diff
@@ -0,0 +1,58 @@
+# Define the root of the local git clone for the common rules to be able 
+# know where they are running from.
+REPOROOT=../../../..
+# Include a library of common .transform.* targets which most
+# transforms should be able to reuse.  However, feel free
+# to override/redefine the rules below. 
+include $(REPOROOT)/transforms/.make.transforms
+
+TRANSFORM_NAME=doc_quality
+# $(REPOROOT)/.make.versions file contains the versions
+DOCKER_HOME_DIR=/home/ray
```

Comment:
What is this here for?

Reply:
It was for mounting the large model's file to container. I' remove.